egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -239,9 +239,9 @@ def process_args(args):
                    If you have GPUs and have nvcc installed, go to src/ and do
                    ./configure; make""")
 
-        run_opts.train_queue_opt = "--gpu 1"
+        run_opts.train_queue_opt = "--gpu 1 --mem 10G"
```

Comment:
I think these queue option changes may be pretty specific to the way your grid infrastructure is set up, e.g. the virtual memory settings or whatever -> not inclined to merge this.